Vostok.ClusterClient.Topology.SD/ServiceDiscoveryClusterProviderSettings.cs

```diff
@@ -0,0 +1,16 @@
+﻿using JetBrains.Annotations;
+using Vostok.Clusterclient.Topology.SD.Transforms;
+
+namespace Vostok.Clusterclient.Topology.SD
+{
+    [PublicAPI]
+    public class ServiceDiscoveryClusterProviderSettings
+    {
+        /// <summary>
+        /// If not set, replicas will be extracted from the IServiceTopology as is.
+        /// Blacklist will be applied in <see cref="ServiceDiscoveryClusterProvider" /> after calling this Transform.
+        /// </summary>
+        [CanBeNull]
+        public IServiceTopologyTransform ServiceTopologyTransform { get; set; }
```

Comment:
All of these transforms only retain their internal state withing a single `ClusterClient` instance (won't work for clients that use `new Client()` for every request). Was this an explicit design decision?